### ISSUE TITLE
There's a way to restore that backups ?

### ISSUE THREAD

**Issue by MHarlock** [caseychu/spotify-backup#54]
Hi there is a way to restore into my spotify account the backup ?

**Comment by WhyNotHugo** [comment]
No, not right now. Spotify's API seems to support writing all the information back, so it's feasible to do it. But nobody has implemented this yet.

**Comment by NicoWeio** [comment]
Duplicate of #40
